<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayInformationOptionsFrame</class>
 <widget class="QFrame" name="DisplayInformationOptionsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>612</width>
        <height>519</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxAutoFill">
         <property name="title">
          <string>Automatic fill empty field with the following data</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="frame">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="labelName">
               <property name="text">
                <string>Author</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="5">
              <widget class="QLineEdit" name="lineEditAuthorName">
               <property name="frame">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QLabel" name="labelOwner">
               <property name="text">
                <string>Owner</string>
               </property>
              </widget>
             </item>
             <item row="13" column="1" colspan="5">
              <widget class="QLineEdit" name="lineEditOwner">
               <property name="frame">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="14" column="0">
              <widget class="QLabel" name="labelLicense">
               <property name="text">
                <string>License</string>
               </property>
              </widget>
             </item>
             <item row="14" column="1" colspan="5">
              <widget class="QLineEdit" name="lineEditLicense">
               <property name="frame">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="1" rowspan="6" colspan="4">
              <widget class="QListWidget" name="listWidget">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Link Sample</string>
                </property>
                <property name="checkState">
                 <enum>Unchecked</enum>
                </property>
               </item>
              </widget>
             </item>
             <item row="8" column="5">
              <widget class="QToolButton" name="toolButtonWaypointDeleteLink_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
                 <normaloff>:/icons/gtk/editdelete.png</normaloff>:/icons/gtk/editdelete.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0" rowspan="5">
              <widget class="QLabel" name="labelLink">
               <property name="text">
                <string>Links</string>
               </property>
              </widget>
             </item>
             <item row="7" column="5">
              <widget class="QToolButton" name="toolButtonWaypointEditLink_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
                 <normaloff>:/icons/gtk/format-text-direction-ltr.png</normaloff>:/icons/gtk/format-text-direction-ltr.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="5" rowspan="2">
              <widget class="QToolButton" name="toolButtonWaypointAddLink_3">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
                 <normaloff>:/icons/gtk/object-add.png</normaloff>:/icons/gtk/object-add.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="9" column="5" rowspan="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1" rowspan="2" colspan="5">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="lineEditEMailId">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="frame">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelAt">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>@</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditEMailDomain">
                 <property name="frame">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelEMail">
               <property name="text">
                <string>e-mail</string>
               </property>
              </widget>
             </item>
             <item row="15" column="1" colspan="5">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Add year based on trace's date</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Add today's year</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="15" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Year</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxType">
         <property name="title">
          <string>Waypoint Route and Track type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="4">
           <widget class="QListWidget" name="listWidgetWaypointLinks">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Type</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="toolButtonWaypointAddLink">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
              <normaloff>:/icons/gtk/object-add.png</normaloff>:/icons/gtk/object-add.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="toolButtonWaypointEditLink">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
              <normaloff>:/icons/gtk/format-text-direction-ltr.png</normaloff>:/icons/gtk/format-text-direction-ltr.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="toolButtonWaypointDeleteLink">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
              <normaloff>:/icons/gtk/editdelete.png</normaloff>:/icons/gtk/editdelete.png</iconset>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCatalog">
         <property name="title">
          <string>Catalog</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBoxShowCatalog">
            <property name="text">
             <string>Show catalog when plugin is activated</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GPSBook/icons/gtk/gtk_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupBoxAutoFill</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
